cloud/deploy/deploy_test.go

```diff
@@ -66,7 +66,7 @@ func TestDeployWithoutDagsDeploySuccess(t *testing.T) {
 
 	mockImageHandler := new(mocks.ImageHandler)
 	airflowImageHandler = func(image string) airflow.ImageHandler {
-		mockImageHandler.On("Build", mock.Anything).Return(nil)
+		mockImageHandler.On("Build", mock.Anything, mock.Anything).Return(nil)
```

Comment:
why do we need to add one more argument 🤔 , I don't see any change in the `Build` function arguments in the `docker_image.go`

Reply:
I changed build but went back. weird that these tests aren't failing

Reply:
I'll fix it